client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -23,6 +23,10 @@ const INITIAL_UNIX_EPOCH = '1970-01-01 00:00:00';
 
 const isExpandedBlockEnabled = config.isEnabled( 'jetpack/pro-dashboard-expandable-block' );
 
+export const siteColumnKeyMap: { [ key: string ]: string } = {
```

Comment:
Nitpick: I would add a comment on what this mapping is about.

Reply:
Updated here - https://github.com/Automattic/wp-calypso/pull/74713/commits/a21e93e40267a719acc8a0eb42a82f78b158865b